ribs/archives/_grid_archive.py

```diff
@@ -122,37 +122,18 @@ def _get_index_numba(behavior_values, upper_bounds, lower_bounds,
         index = (behavior_values - lower_bounds) / interval_size * dims
         return index.astype(np.int32)
 
-    # TODO: Update docstring.
-    def get_index(self, behavior_values):
-        """Returns indices of the behavior values within the archive's grid.
-
-        First, values are clipped to the bounds of the behavior space. Then, the
-        values are mapped to cells; e.g. cell 5 along dimension 0 and cell 3
-        along dimension 1.
-
-        The indices can be used to access boundaries of a behavior value's cell.
-        For example, the following retrieves the lower and upper bounds of the
-        cell along dimension 0::
-
-            idx = archive.get_index(...)  # Other methods also return indices.
-            lower = archive.boundaries[0][idx[0]]
-            upper = archive.boundaries[0][idx[0] + 1]
-
-        See :attr:`boundaries` for more info.
+    def index_of(self, measures):
+        # Adding epsilon to behavior values accounts for floating point
+        # precision errors from transforming behavior values. Subtracting
+        # epsilon from upper bounds makes sure we do not have indices outside
+        # the grid.
+        measures = np.minimum(
```

Comment:
Do these numpy operations in the numba function  (`_get_index_numba`) so that they are fast.

Reply:
not including ravel_multi_index right?